core/arch/arm/plat-totalcompute/conf.mk

```diff
@@ -1,6 +1,4 @@
-PLATFORM_FLAVOR ?= tc0
-
-ifeq ($(PLATFORM_FLAVOR),tc0)
+ifneq ($(filter ${PLATFORM_FLAVOR}, tc0 tc1),)
```

Comment:
Minor: other platforms have the arguments swapped, i.e., the empty one first. I suggest doing the same for consistency. Also, better avoid spaces where not strictly necessary (`, tc0`). So:
```make
ifneq (,$(filter ${PLATFORM_FLAVOR},tc0 tc1)
```

Reply:
Thanks for the review! Updated in latest patch.

Reply:
You're welcome. Please don't forget to add my review tag next to you S-o-b.

Reply:
ah yes, sorry about that. Added in latest commit message.